client/src/js/services/ServiceService.js

```diff
@@ -0,0 +1,43 @@
+angular.module('bhima.services')
+.service('ServiceService', ServiceService);
+
+ServiceService.$inject = [ '$http', 'util' ];
+
+function ServiceService ($http, util) {
```

Comment:
In the future, improvement would be to use a base url.  For example,
```js
var baseUrl = '/services/';

// later, in methods
var target = baseUrl.concat(uuid || '');
$http.get(target) // etc...
```
That way, we only need to define the url once, preventing bugs, adding clarity to the service, and allowing us to change the url easily in the future if we decide to.